qiskit/providers/aer/backends/wrappers/bindings.cc

```diff
@@ -2,27 +2,28 @@
 
 #include <pybind11/pybind11.h>
 
+#include "framework/matrix.hpp"
+#include "framework/types.hpp"
 #include "framework/pybind_json.hpp"
 
 #include "simulators/qasm/qasm_controller.hpp"
 #include "simulators/statevector/statevector_controller.hpp"
 #include "simulators/unitary/unitary_controller.hpp"
-
 #include "simulators/controller_execute.hpp"
 
 PYBIND11_MODULE(controller_wrappers, m) {
     m.def("qasm_controller_execute_json", &AER::controller_execute_json<AER::Simulator::QasmController>, "instance of controller_execute for QasmController");
     m.def("qasm_controller_execute", [](const py::object &qobj) -> py::object {
-        return AER::controller_execute<AER::Simulator::QasmController>(qobj);
+        return AERPy::from_result(AER::controller_execute<AER::Simulator::QasmController>(qobj));
```

Comment:
You have the final call on this, but I'd be more explicit about `AERpy` naming. And we better use `UpperCamelCase` for namespaces, so I'd suggest something like:  `AerToPython`

Reply:
Agreed - was stuck between my intuition and the existing AER and JSON namespaces - I went with AerToPy in ff4a6bc